k8s/client.go

```diff
@@ -0,0 +1,59 @@
+package k8s
```

Comment:
Maybe we should move it to `flyteplugins/pluginmachinery/k8s`

Reply:
That makes sense I was in doubt as well. Should we move the config as well? I think it would make sense actually